packages/unchained-client/openapitools.json

```diff
@@ -122,6 +122,19 @@
           "useSingleRequestParameter": true
         }
       },
+      "polygon": {
+        "inputSpec": "https://raw.githubusercontent.com/shapeshift/unchained/344fe7496a302760c40f18ec3bfcba588e0f5a7e/node/coinstacks/polygon/api/src/swagger.json",
```

Comment:
update before merge

Reply:
y, just need to get upstream merged first